Documentation/PropertyWrapper/Parameter.md

```diff
@@ -298,9 +298,9 @@ struct TestWebService: WebService, Component {
 
     var content: some Component {
         Group("api", "birds", birdID) {
```

Comment:
One nit: To my understanding the `@Parameter` property wrapper now conforms to `PathComponent` but not `@PathParameter`.
So either the code is wrong and the projected value needs to be used here or the example is still outdated? What is the intention? Should the `PathParameter` be passed as a path component or an instance of `Parameter`.

Reply:
Other than that I would happily approve ☺️👍

Reply:
You are right, this should be `$birdID` as this is the projected value of a `@PathParameter` and returns a `@Parameter`. I have updated the .md file